packages/core/src/helpers/import.ts

```diff
@@ -235,10 +239,13 @@ export function parseStImport(atRule: AtRule, context: string, diagnostics: Diag
                 word: importObj.defaultExport,
             });
         }
-        if (imports.tagged?.keyframes) {
-            // importObj.keyframes = imports.tagged?.keyframes;
-            for (const [impName, impAsName] of Object.entries(imports.tagged.keyframes)) {
-                importObj.keyframes[impAsName] = impName;
+        if (imports.tagged) {
+            for (const [kind, namedTyped] of Object.entries(imports.tagged)) {
```

Comment:
looking at this...it leaves the option for any tagged import. but we didn't tell the user when it's a not supported one

Reply:
add diagnostics for unknown typed imports (for both import syntax)